datasetconfig.json

```diff
@@ -302,11 +158,12 @@
         "envtype": ["ocean"],
         "enabled": true,
         "help": " Document <a href=\"../data-help/riops-3d.html\" target=\"_new\">link</a>.",
-        "name": "09. RIOPS Forecast 3D - Polar Stereographic",
+        "name": "06. RIOPS Forecast 3D - Polar Stereographic",
         "quantum": "hour",
         "type": "forecast",
         "cache": 6,
         "time_dim_units": "seconds since 1950-01-01 00:00:00",
+	"bathymetry_file_url": "/data/grids/ecc/misc/polar_stereographic/bathy_meter_giops_ps5km.nc",
```

Comment:
This file is for giops, not riops!! As referenced [here](https://github.com/DFO-Ocean-Navigator/Ocean-Data-Map-Project/pull/753#pullrequestreview-541260330), please use this file:
hank:/tank/data/grids/ecc/riops/bathy_meter_riops_ps5km_reformatted-ONav.nc